integration-tests/src/test/resources/playground/test-processor/src/main/kotlin/BuilderProcessor.kt

```diff
@@ -0,0 +1,83 @@
+import com.google.devtools.ksp.processing.CodeGenerator
+import com.google.devtools.ksp.processing.Dependencies
+import com.google.devtools.ksp.processing.KSPLogger
+import com.google.devtools.ksp.processing.Resolver
+import com.google.devtools.ksp.processing.SymbolProcessor
+import com.google.devtools.ksp.symbol.*
+import java.io.File
+import java.io.OutputStream
+
+fun OutputStream.appendText(str: String) {
+    this.write(str.toByteArray())
+}
+class BuilderProcessor : SymbolProcessor {
```

Comment:
Do you get symbol resolution when writing these tests? If not, I'd consider moving them to a separate project, and make it depends on `ksp`. Then you can publish these processors to the test repository, and just depend on them from the integration projects.

If there is already IDE support, feel free to ignore my comment.

Reply:
Good point. No, IDE doesn't work well with the processors. Will do in a separate patch after supporting specifying a single processor in a shared jar.